raaghu-elements/src/rds-select-list/rds-select-list.tsx

```
@@ -147,71 +190,92 @@ const RdsSelectList = (props: RdsSelectProps) => {
+        }
+   
+        event.stopPropagation();
+      };
+   
+      const handleDoubleClick = () => {
+        // Always clear all on double click
+        setSelectedValue([]);
+        props.onChange?.([]);
+      };
+   
     return (
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)